DependencyInjection/DependencyInjection.csproj

```diff
@@ -102,14 +102,12 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\#packages\Endjin.Core.Composition.2.2.0.45\lib\Net45\EndjinCoreContainerNet45.dll</HintPath>
     </Reference>
-    <Reference Include="Grace, Version=3.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\#packages\Grace.3.1.0.0\lib\portable-net45+win+wpa81+wp80\Grace.dll</HintPath>
+    <Reference Include="Grace, Version=5.1.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\#packages\Grace.5.1.0\lib\net45\Grace.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Grace.MVC4, Version=3.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\#packages\Grace.MVC4.3.1.0.0\lib\net45\Grace.MVC4.dll</HintPath>
+    <Reference Include="Grace.MVC4, Version=5.1.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\#packages\Grace.MVC4.5.1.0\lib\net45\Grace.MVC4.dll</HintPath>

```

Comment:
Seems like change in packages.config is missing from the commit (no problem though, I'll update right now).